internal/pkg/service/stream/storage/node/readernode/sliceupload/operator.go

```diff
@@ -179,26 +191,28 @@ func (o *operator) checkSlice(ctx context.Context, data *sliceData) {
 }
 
 func (o *operator) uploadSlice(ctx context.Context, volume *diskreader.Volume, data *sliceData) {
-	// Get slice statistics
-	stats, err := o.statistics.SliceStats(ctx, data.Slice.SliceKey)
-	if err != nil {
-		o.logger.Errorf(ctx, "cannot get slice statistics: %s", err)
-		return
-	}
-
+	var err error
 	// Empty slice does not need to be uploaded to staging. Just mark as uploaded
-	if stats.Local.RecordsCount != 0 {
-		o.logger.Infof(ctx, `uploading slice %q`, data.Slice.SliceKey)
-		// Use plugin system to upload slice to stagin storage. Set is an in-progress upload
+	if !data.Slice.LocalStorage.IsEmpty {
+		// Get slice statistics
+		var stats statistics.Aggregated
+		stats, err = o.statistics.SliceStats(ctx, data.SliceKey)
+		if err != nil {
```

Comment:
Do we need var stats statistic.Aggregated fo this case?

Can't we use stats,err:=o.statistics.SliceStats(ctx, data.SliceKey) and remove var err error?

Reply:
In original code it is done like that. I would like linter to take care of such definitions. Can you check whether it is possible to enforce this sort of sorthanded definitions checker? :slightly_smiling_face:

Reply:
@jachym-tousek-keboola Ohh sorry, I didn't notice that there aren't `returns`.